app/pages/get-involved/call-for-projects.cjsx

```diff
@@ -40,11 +40,11 @@ counterpart.registerTranslations 'en',
+    You can see examples of our current text transcription projects at [Zooniverse.org](http://zooniverse.org/). These include [AnnoTate](https://anno.tate.org.uk/#/), where volunteers transcribe twentieth-century British artists’ sketchbooks and papers; [Decoding the Civil War](https://www.decodingthecivilwar.org), in which volunteers transcribe telegrams from the American Civil War, and [Shakespeare’s World](https://www.shakespearesworld.org/#/), where volunteers transcribe documents produced in and around Shakespeare’s lifetime (sixteenth and seventeenth century). The resulting datasets will be incorporated into the Tate and Folger Shakespeare Library’s respective content management systems (CMS), and will also be used for research by humanities specialists. We are particularly keen to work on projects where the data can be incorporated into an existing CMS and used for research purposes.
 
     ### Project Selection
 
-    Using our free [Project Builder](/lab) you can now build your own Zooniverse project with a set of tried and tested tools such as questions (yes/no; multiple choice) and region marking or drawing tools. However, while we have created transcription projects such as AnnoTate and Shakespeare’s World, we have not made text transcription tools available on the free Project Builder. Before we build this functionality into the Project Builder we are seeking one or more transcription projects that will help us test and expand the remit of our text transcription tools. These projects need not replicate the functionality of AnnoTate or Shakespeare’s World. They could involve letter-by-letter transcription using a bespoke keyboard, as in Ancient Lives, the first Zooniverse transcription project, which is now retired.
+    Using our free [Project Builder](/lab) you can now build your own Zooniverse project with a set of tried and tested tools such as questions (yes/no; multiple choice) and region marking or drawing tools. More recently we’ve released transcription transcription tools, but are keen to expand their functionality. To that end, we are seeking one or more transcription projects that will help us test and expand the remit of our text transcription tools. These projects need not replicate the functionality of AnnoTate, Decoding the Civil War or Shakespeare’s World. They could involve letter-by-letter transcription using a bespoke keyboard, as in Ancient Lives, the first Zooniverse transcription project, which is now retired.
```

Comment:
Typo in the sentence: `More recently we’ve released transcription transcription tools`